{
  
image_topic: "/camera/color/image_raw",        # Set the image topic

camera_info_topic: "/camera/color/camera_info", # Set the camera info topic

camera_tf: "camera_link",              # Set the camera frame

object_name: "box1_3",                       # Set the object to be detected

reset_bb: True,                            # Set the 2d bounding box manually

depth_enable: False,                        # Enable depth image

depth_topic: "/camera/aligned_depth_to_color/image_raw",  # Set the depth image topic

depth_info_topic: "/camera/aligned_depth_to_color/camera_info", # Set the depth camera info topic

reinitThreshold: 0.1,                       #Reinit threshold for init and track service

refilterThreshold: 0.5,                       #Filter threshold for filter poses

buffer_size: 20                            #Buffer size for filter poses

}